task-2/src/components/Formik/AddMovie/index.js

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from 'react';
+
+import { useDispatch } from 'react-redux';
+
```

Comment:
Consider reviewing empty line between import groups in across the app.
E.g. "react" and "react-redux" are both external to your project libraries, so there's no need to have an empty line between them and it's better to keep them grouped together.